backend/src/repositories/BlocksRepository.ts

```diff
@@ -288,6 +288,7 @@ class BlocksRepository {
       LIMIT 10`;
 
     try {
+      console.log(query, params);
```

Comment:
Use debug.log for this and add a TODO to remove it later

Reply:
Nah I just forgot it :), deleting